File: ChatGPT/ChatGPT/go/fork.go
package main

import (
	"fmt"
	"os/exec"
)

func main() {
	// 실행할 명령어와 인수 설정
	cmd := exec.Command("ls", "-l")

	// 자식 프로세스 실행 및 결과 기다리기
	output, err := cmd.Output()
	if err != nil {
		fmt.Println("Error executing command:", err)
		return
	}

	// 자식 프로세스의 출력 결과 출력
	fmt.Println(string(output))
}

File: ChatGPT/ChatGPT/php/fork.php
<?php

// ls 명령어 실행
exec('/bin/ls -l', $output, $return_var);

// 명령어 실행 결과 출력
if ($output) {
    echo "Output:\n";
    foreach ($output as $line) {
        echo $line . "\n";
    }
}
// 에러 메시지 출력
if ($return_var !== 0) {
    echo "Error: Command execution failed\n";
}

// 종료 코드 출력
echo "Exit code: " . $return_var . "\n";

?>
